.github/CODEOWNERS

```diff
@@ -96,10 +96,14 @@ ui/pages/settings/security-tab/profile-sync-toggle      @MetaMask/identity
 
 app/scripts/lib/snap-keyring         @MetaMask/accounts-engineers
 
-# Swaps team to own code for the swaps folder
+# Swaps-Bridge team to own code for the swaps folder
 ui/pages/swaps                                        @MetaMask/swaps-engineers
 app/scripts/controllers/swaps                         @MetaMask/swaps-engineers
 
+# Swaps-Bridge team to own code for the bridge folder
+**/bridge/**                                          @MetaMask/swaps-engineers
+**/bridge-status/**                                    @MetaMask/swaps-engineers
```

Comment:
Nit: remove 1 space before `@`

Reply:
Done in [7aca15d](https://github.com/MetaMask/metamask-extension/pull/29029/commits/7aca15d8a8fbadb2676244f9fa8cdfb277e28780)